lighty-modules/lighty-restconf-nb-community/src/main/java/io/lighty/modules/northbound/restconf/community/impl/util/RestConfConfigUtils.java

```diff
@@ -48,6 +48,7 @@ public final class RestConfConfigUtils {
     public static final int MAXIMUM_FRAGMENT_LENGTH = 0;
     public static final int IDLE_TIMEOUT =  30000;
     public static final int HEARTBEAT_INTERVAL = 10000;
+    private static String restconfConfigEndpointElementName = "restconf";
```

Comment:
final - and probably we have it already somewhere

Reply:
resolved, completlly reworked